.eslintrc.json

```diff
@@ -9,10 +9,16 @@
     "sourceType": "module"
   },
   "rules": {
+    "brace-style": [
+      "error",
+      "stroustrup"
+    ],
     "indent": [
       "error",
       2,
-      {"SwitchCase": 1}
+      {
+        "SwitchCase": 1
+      }
```

Comment:
Is there a reason you’re expanding all these? I think what I’m generally going for here is equivalent of:

```js
"object-curly-newline": [
  "warn",
  {"multiline": true}
]
```

Reply:
That wasn't intentional. I've reverted those changes